### ISSUE TITLE
FREEBSD -  pstuil dependency too high

### ISSUE THREAD

**Issue by bitclick** [gilesknap/gphotos-sync#443]
## Summary

* OS: FREEBSD 11
* Architecture: 64bit
* Psutil version: 5.8.0
* Python version: Python 3.8.12
* Type:  installation

## Description
- dependency for psutil is higher than neccessary (5.9.4)
- Freebsd 11 ships with python 3.8 which is supported. this comes with psutil 5.8.0 
  - pip psutil cannot be installed because of an upstream bug with python headers (resolved in later Freebsd releases)
  - Freebsd 11 is out of support so a fix will likely not come
- this project runs fine with psutil 5.8.0  when installed manually  (tested with symlinked distro-psutil into the venv `site-packages`)

## possible fix
- lower the required version of psutil to 5.8.0
- pip installation should be working fine in freebsd 11 

## comment
- freebsd 11 is old but it still runs on some hosts with a lot of jails for automation
- with this bug, it is currently not working, though the python-version 3.8 is good enough
- fixing this should be a low-hanging fruit

**Comment by gilesknap** [comment]
@bitclick thanks for the report.

I'm pretty sure I looked at this before and struggled to get a set of dependencies that work for BSD.

If you clone the repo, make the change you suggest and `pip install -e .` does it work??

**Comment by gilesknap** [comment]
closing as no response. Please add to the closed issue if you still want help.

**Comment by bitclick** [comment]
indeed, it works with 

`commit b50baadcd769c09f32b5f9c1020dc3b75540c6f1 (HEAD, tag: 3.2.5)`

and changed dependency

```
diff --git a/pyproject.toml b/pyproject.toml
index a8b92f6..738d6d9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -19,7 +19,7 @@ dependencies = [
     "exif",
     "appdirs",
     "pyyaml",
-    "psutil",
+    "psutil == 5.8",
     "google_auth_oauthlib",
 ] # Add project dependencies here, e.g. ["click", "numpy"]
 dynamic = ["version"]
```

then `pip install -e .`

syncs fine, no problems

**Comment by gilesknap** [comment]
cool.

Will take a look at making this change - but I'm not going to be able to do so until late Sept.

**Comment by bitclick** [comment]
yeah no need to hurry i think.. it's documented here and very few people should be affected
